src/kcm/ExceptionListWidget.cc

```
@@ -0,0 +1,298 @@
+{
+    const QModelIndex index = m_listView->currentIndex();
+    if (!index.isValid()) {
+        return;
+    }
+
+    const auto current = m_model->get(index.row());
+    if (!current) {
+        return;
+    }
+
+    InternalSettingsPtr workingCopy(new InternalSettings());
```

Clang (14.0.6)

[warning] variable 'workingCopy' is not initialized

(cppcoreguidelines-init-variables)